src/main.cpp

```diff
@@ -369,7 +376,7 @@ int main(int argc, char**argv)
                                                    "  smbx2  - Disables all features and bugfixes except fixed at SMBX2\n"
                                                    "  smbx13 - Enforces the full compatibility with the SMBX 1.3 behaviour\n"
                                                    "\n"
-                                                   "  Note: If speed-run mode is set, the compatibility level will be overridden by the speed-run mode",
+                                                   "  Deprecated: acts as an alias for speed-run mode. Will be overridden if speed-run mode is set.",
                                                     false, "modern",
                                                    "modern, smbx2, smbx3",
                                                    cmd);
```

Comment:
The main reason that I added the separated compat level parameter is enabling the compatibility mode without forcing user to enable the speed-runner specific mode that adds the mode indication and speedrun timer. That's the main difference of the compat-level and speed-run mode.

Reply:
EDIT: Ouch, I forgot that you implemented the ability of user to choose the compat mode in the menu per episode :eyes: